package-res/resources/web/pentaho/data/filter/And.js

```diff
@@ -56,6 +56,11 @@ define([
    *  var combination1 = new And([sales12k, inStock]);
    *  var data1 = combination1.apply(data); //data1.getValue(0, 0) === "A"
    * });
+   *
+   * @description Creates an `And` filter given a property name and an array of values to be contained in the set.
```

Comment:
No.
Creates an `And` filter that performs the intersection of a list of filters.
Copy-paste from IsIn gone wrong?